package.json

```diff
@@ -129,86 +129,88 @@
     "gulp-shell": "^0.5.2",
-    "husky": "^3.0.9",
+    "husky": "^3.1.0",
     "intercept-stdout": "^0.1.2",
-    "mocha": "^6.2.0",
-    "nock": "^11.0.7",
+    "mocha": "^6.2.2",
+    "nock": "^11.7.2",
     "parse-link-header": "~1.0.1",
-    "pkg": "^4.4.0",
+    "pkg": "^4.4.2",
     "prettier": "^1.19.1",
-    "publish-release": "^1.6.0",
+    "publish-release": "^1.6.1",
     "resin-lint": "^3.1.1",
     "rewire": "^3.0.2",
-    "sinon": "^7.4.1",
-    "ts-node": "^8.1.0",
-    "typescript": "^3.4.5"
+    "sinon": "^7.5.0",
+    "ts-node": "^8.6.2",
+    "typescript": "^3.7.5"
   },
   "dependencies": {
-    "@oclif/command": "^1.5.12",
+    "@oclif/command": "^1.5.19",
     "@oclif/errors": "^1.2.2",
     "@resin.io/valid-email": "^0.1.0",
     "@zeit/dockerignore": "0.0.3",
     "JSONStream": "^1.0.3",
     "ansi-escapes": "^2.0.0",
     "any-promise": "^1.3.0",
     "archiver": "^2.1.0",
-    "balena-config-json": "^2.1.0",
-    "balena-device-init": "^5.0.0",
-    "balena-device-status": "^3.1.2",
-    "balena-image-manager": "^6.1.0",
-    "balena-preload": "^8.2.1",
-    "balena-sdk": "^12.16.0",
-    "balena-settings-client": "^4.0.0",
-    "balena-sync": "^10.0.3",
+    "balena-config-json": "^2.1.1",
+    "balena-device-init": "^5.0.1",
+    "balena-device-status": "^3.2.0",
+    "balena-errors": "^4.2.1",
+    "balena-image-manager": "^6.1.1",
+    "balena-preload": "^8.3.0",
+    "balena-sdk": "^12.21.1",
+    "balena-settings-client": "^4.0.2",
+    "balena-sync": "^10.1.0",
     "bash": "0.0.1",
-    "bluebird": "^3.5.4",
+    "bluebird": "^3.7.2",
     "body-parser": "^1.19.0",
-    "capitano": "^1.9.0",
+    "capitano": "^1.9.1",
     "chalk": "^2.3.0",
-    "chokidar": "^2.0.4",
+    "chokidar": "^2.1.8",
     "cli-truncate": "^1.1.0",
     "coffeescript": "^1.12.6",
     "color-hash": "^1.0.3",
     "columnify": "^1.5.2",
     "common-tags": "^1.7.2",
     "denymount": "^2.3.0",
-    "docker-modem": "^2.0.2",
+    "docker-modem": "^2.0.4",
     "docker-progress": "^4.0.0",
     "docker-qemu-transpose": "^1.0.2",
     "docker-toolbelt": "^3.3.7",
     "dockerode": "^2.5.8",
-    "ejs": "^2.5.7",
-    "etcher-sdk": "^2.0.14",
+    "ejs": "^3.0.1",
+    "etcher-sdk": "^2.0.16",
     "event-stream": "3.3.4",
     "express": "^4.13.3",
     "fast-boot2": "^1.0.9",
     "global-tunnel-ng": "^2.1.1",
     "hasbin": "^1.2.3",
     "humanize": "0.0.9",
-    "ignore": "^5.1.1",
+    "ignore": "^5.1.4",
     "inquirer": "^3.1.1",
     "is-elevated": "^3.0.0",
     "is-root": "^2.1.0",
     "js-yaml": "^3.13.1",
     "klaw": "^3.0.0",
     "livepush": "^2.0.6",
-    "lodash": "^4.17.4",
+    "lodash": "^4.17.15",
     "minimatch": "^3.0.4",
-    "mixpanel": "^0.10.1",
+    "mixpanel": "^0.10.3",
     "mkdirp": "^0.5.1",
     "moment": "^2.24.0",
-    "moment-duration-format": "^2.2.2",
+    "moment-duration-format": "^2.3.2",
     "mz": "^2.7.0",
     "node-cleanup": "^2.1.2",
-    "oclif": "^1.13.1",
+    "node-unzip-2": "^0.2.8",
```

Comment:
This is identical to `"unzip2": "balena-io-library/node-unzip-2#v0.2.8"` in terms of contents and is just pointing it to the npm version to avoid the need for the git dep